### DISCUSSION TITLE
Issue installing security onion 2.3

### DISCUSSION THREAD

**Discussion by uskwarrior** [Security-Onion-Solutions/securityonion#2767] [Unsupported Versions] [answered]
Hello Dear folks,

I am a student in Cybersecurity and I wanted to use security onion to setup my security lab. I had used [security onion 16.04](https://github.com/Security-Onion-Solutions/security-onion) and had no issues whatsoever with the only issue being it doesn't have the security console option which has been introduced in [Security onion 2.3](https://github.com/Security-Onion-Solutions/securityonion/blob/master/VERIFY_ISO.md). I tried installing the security onion 2.3 where the setup and installation gets completed however each time I am asked to reboot and after reboot I see the setup page once again and I need to re-do the installation. I have now done this process for 5-6 times with no luck. I used the recommended configurations (12G Memory and 100Gb hard disk) for the installation but it still failed. Can someone please help? Is it not supported on AMD Ryzen 7 processor ? Am I doing something wrong ?

![image](https://user-images.githubusercontent.com/78109598/106049905-abf0d900-60b4-11eb-9c12-52eaedb77dd8.png)

![2](https://user-images.githubusercontent.com/78109598/106049965-b612d780-60b4-11eb-82e8-5125ff66b8dc.png)
![3](https://user-images.githubusercontent.com/78109598/106049966-b612d780-60b4-11eb-8f32-cdff671990bb.png)
![1](https://user-images.githubusercontent.com/78109598/106049967-b612d780-60b4-11eb-893e-e667c2d3f075.png)

**Comment by uskwarrior** [comment]
Here are the logs
[VBoxHardening.log](https://github.com/Security-Onion-Solutions/securityonion/files/5882913/VBoxHardening.log)
[VBox.log](https://github.com/Security-Onion-Solutions/securityonion/files/5882914/VBox.log)

**Comment by TOoSmOotH** [comment, marked as answer]
Disconnect the virtual cdrom or select it to boot off your virtual harddrive.

**Reply by uskwarrior** [reply]
Thank you so much @TOoSmOotH It worked. I have a few ,more questions if I may ask

1. Post installation, when I logged into the security onion, I see there is no GUI and on the VM, I keep getting these console message continuously interrupting me each time I am  trying to type a command. How can I disable these console messages?

![image](https://user-images.githubusercontent.com/78109598/106189760-b5427a00-6176-11eb-97aa-f6e4c576ead8.png)

2. Since accessing the GUI on the server hasn't been possible, I connected another VM to the security onion VM and allowed port 80 and 443 using "so-allow" however I am unable to access the security console. When performing nmap i see the ports are still closed. Is there another process to maybe start the service or allow ports.?

**Comment by KrisjonHanson** [comment]
You can install the analyst tools on your management server with the so-analyst-install script.

https://docs.securityonion.net/en/2.3/analyst-vm.html
